homeassistant/components/ld2410_ble/binary_sensor.py

```diff
@@ -0,0 +1,81 @@
+"""LD2410 BLE integration binary sensor platform."""
+
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import LD2410BLE, LD2410BLECoordinator
+from .const import DOMAIN
+from .models import LD2410BLEData
+
+ENTITY_DESCRIPTIONS = [
+    BinarySensorEntityDescription(
+        key="is_moving",
+        device_class=BinarySensorDeviceClass.MOTION,
+        has_entity_name=True,
+        name="Motion",
+    ),
+    BinarySensorEntityDescription(
+        key="is_static",
+        device_class=BinarySensorDeviceClass.OCCUPANCY,
+        has_entity_name=True,
+        name="Occupancy",
+    ),
+]
```

Comment:
MINOR: This could be a tuple since it never changes

Reply:
Noted. I might change it in a subsequent MR.